lib/poker_hands/card.rb

```diff
@@ -0,0 +1,10 @@
+class Card
+    def initialize(rank, suit)
+        @rank = rank
+        @suit = suit
+    end
+
+    def get_card
```

Comment:
Lots of objects in Ruby have a builtin `to_s` (to string) method, which is used for getting a string representation of the object.

If we wanted to adhere to that convention, we could name this method `to_s`

```
> card = Card.new(1, 'H')
> card.to_s
"1H"
```

This would also help communicate that we can expect the method to return a string.